### PR TITLE
Update comments and steps

### LINKED ISSUE #161: Clean-up setup.sh comments and steps.
**Branch** master

**Target** 0.15.x

**Current Behavior** Some steps on the script must be removed or changed.

```
# 6. Make ka-lite repo production-ready (delete .KALITE_SOURCE_DIR, etc).
# 9. Create the `<Xcode_Resources>/ka-lite/kalite/local_settings.py` based on `local_settings.default`.
```

**Expected Behavior** The comments on the script must be updated accordingly.

### PR DESCRIPTION
**Pull request by arceduardvincent**
Hi @cpauya and @aronasorman  this fixes #161 

**Steps updated and removed:**
>1. Include the steps to download the assessment.zip and copy it to `Xcode Resources` folder.
>1. Will not include the `ka-lite` folder to copy in the Xcode Resources folder
>1. Remove step on creating the `local_settings.py` based on `local_settings.default`.
>1. Remove this step " `Make ka-lite repo production-ready (delete .KALITE_SOURCE_DIR, etc).".
 it is not necessary anymore. 
>1. Add this step "Uninstall pyyaml so it's not included in the .dmg to build."

### CODE REVIEW

File: osx/setup.sh [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
+# 4. Download PyRun thru `install-pyrun` script.
+# 5. Download KA-Lite zip based on develop branch and extract KA-Lite and move into `ka-lite` folder.
+# 6. Install the `ka-lite-static` by running `pyrun setup.py install` inside the `ka-lite` directory.
 # 7. Run pyrun-2.7/bin/pip install -r ka-lite/requirements.txt
-# 8. Run `bin/kalite manage compileymltojson`, needs `pyrun/pip install pyyaml==3.11`
-# 9. Create the `<Xcode_Resources>/ka-lite/kalite/local_settings.py` based on `local_settings.default`.
-# 10. Copy the `ka-lite` and `pyrun` folders to the Xcode Resources folder.
-# 11. Build the Xcode project to produce the .app.
-# 12. Build the .dmg.
+# 8. Install requirements for sphinx
```
Comment (cpauya) [inline osx/setup.sh]: Please change this to mention that we are building the docs using `sphinx-build`.

File: osx/setup.sh [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
+# 9. Run `bin/kalite manage compileymltojson`, needs `pyrun/pip install pyyaml==3.11`
+# 10. Uninstall pyyaml so it's not included in the .dmg to build
+# 11. Copy `pyrun` folders to the Xcode Resources folder.
```
Comment (cpauya) [inline osx/setup.sh]: I think there's only one `pyrun` folder here.

File: osx/setup.sh [hunk 3]
```diff
@@ -51,6 +52,8 @@ pushd `dirname $0` > /dev/null
 SCRIPTPATH=`pwd`
 popd > /dev/null
 
+# Create temporary directory
+((STEP++))
```
Comment (cpauya) [inline osx/setup.sh]: Remove this, `STEP` is already initialized to value of 1.

**Comment by arceduardvincent** [comment]
Hello @cpauya I was able to update the setup.sh as your requested above.

**Comment by cpauya** [comment]
@arceduardvincent merge conflicts

File: osx/setup.sh [hunk 4]
```diff
@@ -240,26 +245,26 @@ if [ $? -ne 0 ]; then
...
 cd $KA_LITE_DIR
 if [ -d "$KA_LITE_DIR" ]; then
     echo "Install npm.."
-    npm install
-    ulimit -n 2560
-    node build.js
-    cd $KA_LITE_DOCS_DIR
-    "$PYRUN_SPHINX_BUILD" -b html -d _build/doctrees   . _build/html
-    cp -R -v "$KA_LITE_DOCS_DIR" "$PYRUN_DIR/share/kalite"
+        npm install
```
Comment (cpauya) [inline osx/setup.sh]: @arceduardvincent Indentation is off here.

File: osx/setup.sh [hunk 5]
```diff
@@ -253,13 +256,15 @@ if [ -d "$KA_LITE_DIR" ]; then
     npm install
     ulimit -n 2560
     node build.js
-    cd $KA_LITE_DOCS_DIR
-    "$PYRUN_SPHINX_BUILD" -b html -d _build/doctrees   . _build/html
-    cp -R -v "$KA_LITE_DOCS_DIR" "$PYRUN_DIR/share/kalite"
+    cd $KA_LITE_DIR/docs
```
Comment (cpauya) [inline osx/setup.sh]: Use `$KA_LITE_DOCS_DIR` instead of hard-coding again.

**Pull request merged**